.github/workflows/acceptance-tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [16.x]
```

Comment:
let's remove matrixes altogether if we're only ever using a single node version. let's provide the node version in the setup-node action inputs, and let's also use 16 instead of 16.x